src/fn/fn-message-strip.scss

```diff
@@ -0,0 +1,101 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-message-strip;
+
+.#{$block} {
+  @include fn-reset();
+  @include fn-flex();
+  @include fn-set-paddings-x(0.5rem, 0.5rem);
+
+  align-items: center;
+  background-color: $fn-color-blue-50;
+  color: $fn-color-grey-700;
+  min-height: 1.75rem;
+  min-width: 20.5rem;
+  width: fit-content;
+  border-radius: $fn-border-radius-4;
+
+  &__text {
+    @include fn-reset();
+
+    color: inherit;
+    max-width: 95%;
+  }
+
+  &__icon {
```

Comment:
If you want to avoid the usage of !important you can use `.#{$block}__icon` instead of `&__icon`. This will enforce the specificity. And then you can use the mixin for setting the margin-right `@include fn-set-margin-right(0.25rem)`